src/rpc/rpc_holder.h

```diff
@@ -224,9 +224,13 @@ class rpc_holder
         dsn_rpc_forward(dsn_request(), addr);
     }
 
-    inline void set_auto_reply(bool auto_reply) { _i->auto_reply = auto_reply; }
+    [[nodiscard]] inline bool is_auto_reply() const { return _i->auto_reply; }
 
-    inline void enable_auto_reply() { set_auto_reply(true); }
+    inline void set_auto_reply(bool auto_reply) const { _i->auto_reply = auto_reply; }
```

Comment:
Can it be private?

Reply:
Yes, currently we've never called `set_auto_reply()` externally. So it could be made private.